### PR TITLE
Basic script for populating db

### PR DESCRIPTION
**Pull request by riyajaggi**
### Summary 

This pull request is the first step towards populating the teams db in idol. This makes use of existing member documents and adds them to the team collections based on their role and subteam.

### Test Plan 

Ensure all team endpoints are working correctly after populating the db

### Notes 

This still needs some work.

### Breaking Changes 

- no change in schema
- need to ensure all teams are updated correctly

### CODE REVIEW

File: backend/data/firebase.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require('dotenv').config({path:'/backend/data/.env'})
+
+import admin = require('firebase-admin');
+
+const serviceAccount = require('backend/resources/cornelldti-idol-firebase-adminsdk-ifi28-9aaca97159.json');
```
Comment (SamChou19815) [inline backend/data/firebase.ts:5]: could you fix these linter errors and warnings first

File: backend/data/populate-teams.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      } else {
+        teamData = {
+            uuid: teamData.uuid,
+            name: teamData.name, 
+            leaders: teamData.leaders,
+            members: teamData.members.add(/members/+email)
+        };
+      }
+    //   db.doc(`teams/${teamData.uuid}`).update(teamData);
+    console.log(teamData);
```
Comment (SamChou19815) [inline backend/data/populate-teams.ts:34]: this function doesn't seem to do anything at the moment, due to this db code being commented out

**Review by SamChou19815 — CHANGES_REQUESTED** [review]

**Comment by henry-li-06** [comment]
I'm not sure if we really need to keep the script to populate `teams` collection. We theoretically won't ever need to mass populate the collection again since teams can be added/deleted/edited from the _Edit Teams_ page. Thoughts @SamChou19815?

**Comment by SamChou19815** [comment]
@henry-li-06 good point. Also why are we keeping the teams collection if the team information can already be inferred from the subteam field?

**Comment by henry-li-06** [comment]
@SamChou19815 I guess we don't necessarily need the teams collection since the entirety of the collection can be derived from the members collection and the relationship between members and teams adds some complexity to our data. But at the same time I think operations on teams would be less computationally wasteful if we leave the collection (ex. don't need to go through all members to delete a team).

**Comment by SamChou19815** [comment]
@henry-li-06 Given that we are only deal with data set with size around 100, I think that kind of performance benefits are negligible, but the cost of syncing the data in two collections are real. That being said, I'm ok with #103 merged if breaks are fixed, since we need to fix the subteam issue first.

**Pull request closed without merging**